package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "react-typescript-boilerplate",
+  "version": "0.0.1",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\"",
+    "start": "webpack-dev-server --config config/webpack/webpack.dev.js",
+    "build:clean": "rimraf ./dist",
```

Comment:
Can use CleanWebpackPlugin for this. So we dont need to clean the dist manually. We can use both